src/styles/components/_inline-title-edit.scss

```diff
@@ -48,7 +48,7 @@
 		}
 
 		& + .c-floating-error {
-			margin-top: $g-spacer-unit;
+			bottom: -3rem;
```

Comment:
Negative margins can cause problems, is there way to not use it? Add top margin?